src/main/solution_space/solution_space_handler.py

```diff
@@ -19,6 +20,14 @@ def __get_column_value(solutions: pd.DataFrame, index: int, column=None) -> str:
     return solutions[column].iloc[index]
 
 
+def __get_column_unique_value(solutions: pd.DataFrame, column=None) -> str:
```

Comment:
The function can return not only str:  the `unique()` function saves type for values in column